### ISSUE TITLE
Documentation use lower priced examples

### ISSUE THREAD

**Issue by taipignas** [hashicorp/terraform-provider-azurerm#16482]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Change example sku from BC_Gen5_2 to S0 or Basic. 900€ documentation example is a bit much.

```
resource "azurerm_mssql_database" "test" {
  name           = "acctest-db-d"
  server_id      = azurerm_mssql_server.example.id
  collation      = "SQL_Latin1_General_CP1_CI_AS"
  license_type   = "LicenseIncluded"
  max_size_gb    = 4
  sku_name       = "Basic"
```

https://docs.microsoft.com/en-us/powershell/module/az.sql/get-azsqlserverserviceobjective?view=azps-7.4.0

![image](https://user-images.githubusercontent.com/55182635/164384143-b7cada1a-b954-49c2-9498-60cba0fa9e22.png)

### New or Affected Resource(s)/Data Source(s)

azurerm_mssql_database

### Potential Terraform Configuration

```hcl
provider "azurerm" {
  features {}
}

resource "azurerm_resource_group" "example" {
  name     = "example-resources"
  location = "West Europe"
}

resource "azurerm_storage_account" "example" {
  name                     = "examplesa"
  resource_group_name      = azurerm_resource_group.example.name
  location                 = azurerm_resource_group.example.location
  account_tier             = "Standard"
  account_replication_type = "LRS"
}

resource "azurerm_mssql_server" "example" {
  name                         = "example-sqlserver"
  resource_group_name          = azurerm_resource_group.example.name
  location                     = azurerm_resource_group.example.location
  version                      = "12.0"
  administrator_login          = "4dm1n157r470r"
  administrator_login_password = "4-v3ry-53cr37-p455w0rd"
}

resource "azurerm_mssql_database" "test" {
  name           = "acctest-db-d"
  server_id      = azurerm_mssql_server.example.id
  collation      = "SQL_Latin1_General_CP1_CI_AS"
  license_type   = "LicenseIncluded"
  max_size_gb    = 4
  read_scale     = true
  sku_name       = "BC_Gen5_2"
  zone_redundant = true

  extended_auditing_policy {
    storage_endpoint                        = azurerm_storage_account.example.primary_blob_endpoint
    storage_account_access_key              = azurerm_storage_account.example.primary_access_key
    storage_account_access_key_is_secondary = true
    retention_in_days                       = 6
  }


  tags = {
    foo = "bar"
  }

}
```

**Comment by sinbai** [comment]
@taipignas thank you for opening this issue. This issue has been fixed by PR [#16398](https://github.com/hashicorp/terraform-provider-azurerm/pull/16398). It has been released in[ v3.3.0](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/mssql_database) of the terraform provider doc.

**Comment by tombuildsstuff** [comment]
Closing as this has been fixed by #16398
